test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs

```diff
@@ -1913,6 +1922,8 @@ public async Task RestoreCommand_PreservesPackageFilesTimestamps()
                 var dllFileInfo = new FileInfo(dllPath);
                 Assert.True(File.Exists(dllFileInfo.FullName));
                 Assert.Equal(entryModifiedTime, dllFileInfo.LastWriteTime);
+
+                // TODO NK - 
```

Comment:
todos should be resolved before merging.

Reply:
oh nice catch. 

This was failing locally for me, but never failed on the CI. 

Removed.